base/Dockerfile

```diff
@@ -7,7 +7,8 @@ RUN groupadd -r socrata && useradd -r -g socrata socrata
 
 RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && DEBIAN_FRONTEND=noninteractive apt-get -y install \
   curl \
-  python-jinja2
+  python-jinja2 && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
Nit... can you indent this so the list of packages is indented an extra level versus the next command?